README.md

```diff
@@ -8,30 +8,38 @@ This tool can currently be used to update the Black Magic Debug firmware on your
 
 ## Installation
 
+Binary releases for Linux, Mac (x64/ARM) and Windows (x64/ARM) are now available with every
+[release](https://github.com/blackmagic-debug/bmputil/releases). These should work out-of-the-box,
+and do not require manual installation of Windows Driver Kit 8.0 or Rust.
+
+
+Alternately, you can install directly from [crates.io](https://crates.io/crates/bmputil) with cargo.
+
 First install Rust on your computer. Follow the instructions on the [Rust Lang website](https://www.rust-lang.org/tools/install).
 
-Now you can build and install the tool from source.
+Then, install bmputil using `cargo install bmputil`
+
+bmputil on Windows will attempt to automatically setup driver installation on first run.
+This is extra experimental, and will require administrator access on the first run.
+
+## Building from source
```

Comment:
Please insert a new line after this heading (it's a Markdown lint) and remove the extra one from line 15